Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4033,6 +4034,13 @@ void AXObjectCache::updateRelationsIfNeeded()
         // Collect all possible origins, i.e., elements with non-empty relation attributes.
         for (const auto& attribute : relationAttributes()) {
             auto& idsString = element.attributeWithoutSynchronization(attribute);
+            if (idsString.isNull()) {
```

Comment:
We need to remember to fix this code for shadow DOM later.

Reply:
Should we add a FIXME for later?

Reply:
Sure, added.

Reply:
Where is the FIXME?